### ISSUE TITLE
Commitment savings are inaccurate, need help to get the true saving numbers from RI and SP like in the ACM

### ISSUE THREAD

**Issue by saleemmy** [microsoft/finops-toolkit#1003]
![image](https://github.com/user-attachments/assets/83f38527-ce03-4512-8f44-2e6b2d6a622e)

![image](https://github.com/user-attachments/assets/877bdccf-2ab4-4604-9d5d-9440c63917b1)

The savings numbers are not correct.  the commitment savings do not show values. 
since moving to MCA, ACM app not available, I have been struggling to find out the true savings from RI and SP.

**Comment by flanakin** [comment]
This is a known limitation in Cost Management where they do not share list and contracted prices/cost for reservations in MCA accounts, so we can't calculate savings today. We are working on a workaround. Stay tuned...

**Comment by wardjas** [comment]
Hi I was just wondering if the work around has ever been figured out? We just changed from EA to MCA now we are seeing this same issue and having that savings shown was a huge part. Thanks for the work on this.

**Comment by kasimrehman** [comment]
It seems this comment may be relevant: https://github.com/microsoft/finops-toolkit/issues/874#issuecomment-2653292230

**Comment by flanakin** [comment]
@wardjas Yes, we have a workaround to this in FinOps hubs with Data Explorer and the storage reports.

**Comment by peter-mogaka** [comment]
To add to this. After MCA and enabling "add missing prices" savings cost for RI seem off. 

- Storage reports show total savings being way more than effective cost. 
- It shows Zero savings when "add missing prices" is false. 
- 
See screenshots below. Green boxes = EA billing. Red boxes = MCA billing. Purple line- total savings

Using finops v0.11 for hub and reports.

![Image](https://github.com/user-attachments/assets/d513d614-d783-4671-a0db-30a4da467e20)
![Image](https://github.com/user-attachments/assets/6545eb1f-e6e3-40cb-986b-b3c153a754df)

@flanakin any workaround that doesn't involve ADX/ fabric?

**Comment by flanakin** [comment]
@peter-mogaka Which column is that? What do the rows represent? We'd have to drill into the data to investigate. The storage reports don't have a ton of testing given how slow they are to load. I wonder if there's a specific SKU or set of SKUs that may not be joining correctly 🤔 We'd need more info to know for sure.

**Comment by peter-mogaka** [comment]
The columns are effective costs(left), savings(right) per month. See top of the screenshot for naming. What info do you need? Happy to jump on a call if that helps. 

FYI - Top screenshot is for cost savings report: The bottom with a graph is rate optimization report.

**Comment by flanakin** [comment]
Sorry, which savings column are you using? That's what I was asking about. We also need to know what the rows represent to try to reproduce it.

**Comment by peter-mogaka** [comment]
Total savings column (x_TotalSavings) for both reports. Rows are subscriptions. The top screenshot is from the summary tab of the cost summary report.

**Comment by peter-mogaka** [comment]
@flanakin do you have time to jump on call the next few weeks?
